interfaces/cython/cantera/cti2yaml.py

```diff
@@ -0,0 +1,1648 @@
+        :param b:
+            The temperature exponent. Dimensionless. Default: 0.0.
+        :param E:
+            Activation energy. Default: 0.0.
+        :param coverage: For a single coverage dependency, a list with four
+            elements: the species name followed by the three coverage
+            parameters. For multiple coverage dependencies, a list of lists
+            containing the individual sets of coverage parameters. Only used for
+            surface and edge reactions.
+        """
+
+        self._c = [A, b, E]
+
+        if coverage:
+            if isinstance(coverage[0], basestring):
+                self._cov = [coverage]
+            else:
+                self._cov = coverage
+            for cov in self._cov:
+                if len(cov) != 4:
+                    raise InputError("Incorrect number of coverage parameters")
+        else:
+            self._cov = None
+
+    @classmethod
+    def to_yaml(cls, representer, node):
+        out = FlowMap([('A', applyUnits(node._c[0])),
+                       ('b', applyUnits(node._c[1])),
+                       ('Ea', applyUnits(node._c[2]))])
+        return representer.represent_dict(out)
+
+
+class stick(Arrhenius):
+    def __init__(self, *args, **kwargs):
+        """
+        :param motz_wise: 'True' if the Motz & Wise correction should be used,
+            'False' if not. If unspecified, use the mechanism default (set using
+            the functions `enable_motz_wise` or `disable_motz_wise`).
+        """
+        self.motz_wise = kwargs.pop('motz_wise', None)
+        Arrhenius.__init__(self, *args, **kwargs)
+
+
+class reaction(object):
+    """
+    A homogeneous chemical reaction with pressure-independent rate coefficient
+    and mass-action kinetics.
+    """
+    def __init__(self, equation, kf, id='', order='', options=[]):
+        r"""
+        :param equation:
+            A string specifying the chemical equation.
+        :param kf:
+            The rate coefficient for the forward direction. If a sequence of
+            three numbers is given, these will be interpreted as [A, b, E] in
+            the modified Arrhenius function :math:`A T^b exp(-E/\hat{R}T)`.
+        :param id:
+            An optional identification string.
+        :param order:
+            Override the default reaction orders implied by the reactant
+            stoichiometric coefficients. Given as a string of key:value pairs,
+            e.g., ``"CH4:0.25 O2:1.5"``.
+        :param options: Processing options, as described in
+            `Options <https://cantera.org/tutorials/cti/reactions.html#options>`_.
+            May be one or more (as a list) of the
+            following: ``'duplicate'``, ``'negative_A'``,`` 'negative_orders'``,
+            ``'nonreactant_orders'``.
+        """
+        self._e = equation
+        self._order = getComposition(order)
+        self._num = len(_reactions['reactions']) + 1
+        self._id = id
```

Comment:
It would have been better to use a different name, but this name has been baked into the CTI specification for at least a decade and can't be changed now.